lib/central.dart

```diff
@@ -1,14 +1,17 @@
-import 'dart:async';
+abstract class CentralRound {
+  String? get variant;
+  String? get fen;
+  String? get lastMove;
+}
 
 abstract class Central {
```

Comment:
Current interface is prepared in that way that Central would keep Peripheral reference and Peripheral will use Central. Within that there is risk that Central will check peripheral features or variants before devices protocol handshake